### DISCUSSION TITLE
Doom emacs (or otherwise evil-mode users), what are your keybindings?

### DISCUSSION THREAD

**Discussion by kelvie** [karthink/gptel#185] [General]
Just wondering what are some unintrusive keybindings people are using for this.

So far I'm using:

``` elisp
(map! :leader
      (:prefix "a"
       :desc "GPTel Menu" "a" #'gptel-menu
       :desc "GPTel Send" "s" #'gptel-send
       :desc "GPTel Abort" "k" #'gptel-abort
       ))
```

so `SPC a a` to start the menu, or `SPC a s` to send. I'm also considering `l` for LLMs.

**Comment by karthink** [comment]
These are not keybindings exactly, but you might be interested in #189.

**Comment by akhil3417** [comment]
`C-c C-RET` globally for gptel and same for gptel-menu in gptel buffer 

my module for gptel in doom-eMacs see [here](https://github.com/akhil3417/doomemacsonsteroids/blob/master/modules/tools/gptel/config.el)

**Comment by isaacphi** [comment]
This is what I've got:

```
(map! :leader
      (:prefix-map ("l" . "LLM")
       :desc "Send text to model" "RET" #'gptel-send
       :desc "Send text to model with arguments" "l" #'gptel-menu
       :desc "Send text to model with arguments" "s" #'gptel-menu
       :desc "Abort query" "x" #'gptel-abort
       :desc "Chat buffer" "c" #'gptel))
(map! :leader :prefix "o" :desc "LLM Chat buffer" :n "c" #'gptel)
```
